README.md

```diff
@@ -1,3 +1,5 @@
 [![Build Status](https://api.travis-ci.org/MozillaSecurity/lithium.svg)](https://travis-ci.org/MozillaSecurity/lithium)
 [![Build Status](https://ci.appveyor.com/api/projects/status/69f2jn9f4kgjrffj/branch/master?svg=true)](https://ci.appveyor.com/project/MozillaSecurity/lithium/branch/master)
 [![codecov](https://codecov.io/gh/MozillaSecurity/lithium/branch/master/graph/badge.svg)](https://codecov.io/gh/MozillaSecurity/lithium)
+
+See [this page as a start](lithium/doc/using.html).
```

Comment:
This link shows the raw html... we could prepend `https://htmlpreview.github.io/?` to the full url (eg. `https://htmlpreview.github.io/?https://raw.githubusercontent.com/MozillaSecurity/lithium/master/lithium/doc/using-for-firefox.html` .. but using the full url seems wrong to me... it would always show our version for forks as an example.

It's probably better to just convert these to markdown? I can do this quickly if you agree but don't want to. ;)